src/main/java/duke/util/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke.util;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.function.Consumer;
+
+public class TaskList {
+
+    private final ArrayList<Task> list;
+
+    /**
+     * Default constructor for a TaskList.
+     *
+     * @param tasks The ArrayList of Tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        list = tasks;
+    }
+
+    /**
+     * Constructor for an empty TaskList.
+     */
+    public TaskList() {
+        list = new ArrayList<>();
+    }
+
+    /**
+     * Gets the Task with the specified index.
+     *
+     * @param index The index of the Task we want to get.
+     * @return The Task with the specified index.
+     */
```

Comment:
vague naming convention. Perhaps try to use things such as getTask or something that explains what you are trying to get

Reply:
okay I realize its actually okay cos this is tasklist